### PR TITLE
C++/WinRT Template Update

### PR DESCRIPTION
**Pull request by jwoo-msft**
# Description
1. Updated C++/WinRT Template to accept host config as string
2. Changed nuget name from AdaptiveCards.Template to AdaptiveCards.Templating.WinRT to be consistent with the dotnet library.
3. Updated the test app.
4. Updated the signing to fix the build break.

# How Verified
Ran sample app.

### CODE REVIEW

File: source/dotnet/Library/AdaptiveCards.Templating.CSharp.WinRT/AdaptiveCards.Template.csproj [hunk 1]
```diff
@@ -21,10 +21,6 @@
 
   <PropertyGroup>
     <CsWinRTComponent>true</CsWinRTComponent>
-    <SignAssembly>True</SignAssembly>
```
Comment (almedina-ms) [inline source/dotnet/Library/AdaptiveCards.Templating.CSharp.WinRT/AdaptiveCards.Template.csproj:24]: How was this solved? Does it require a matching change in the Azure repo?

File: source/dotnet/Library/AdaptiveCards.Templating.CSharp.WinRT/AdaptiveCards.Template.csproj [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <Authors>Microsoft</Authors>
     <Summary>Adaptive Card Template for WinRT</Summary>
     <Description>This library implements classes for building and serializing adaptive card objects</Description>
-    <PackageId>AdaptiveCards.Template</PackageId>
```
Comment (almedina-ms) [inline source/dotnet/Library/AdaptiveCards.Templating.CSharp.WinRT/AdaptiveCards.Template.csproj:8]: Will this introduce a breaking change or a nuget package name?

**Comment by almedina-ms** [comment]
please rename pull request to match what is the intent of the changes

File: source/dotnet/Library/AdaptiveCards.Templating.CSharp.WinRT/AdaptiveCards.Template.csproj (hunk 2 above)
Comment (jwoo-msft) [reply source/dotnet/Library/AdaptiveCards.Templating.CSharp.WinRT/AdaptiveCards.Template.csproj:8]: yes, it's requested by Widget team

**Review by paulcam206 — APPROVED** [review]

File: source/dotnet/Library/AdaptiveCards.Templating.CSharp.WinRT/AdaptiveCards.Template.csproj (hunk 1 above)
Comment (jwoo-msft) [reply source/dotnet/Library/AdaptiveCards.Templating.CSharp.WinRT/AdaptiveCards.Template.csproj:24]: I didn't necessarily fixed the issue, I took the reference from Paul's OB change.

**Pull request merged**